proto/proto_test.go

```diff
@@ -330,7 +330,8 @@ func TestHasResponseTitle(t *testing.T) {
 		"HTTP/1.1 100 Continue\r\n": true,
 		"HTTP/1.1  \r\n":            false,
 		"HTTP/4.0 100Continue\r\n":  false,
-		"HTTP/4.0 100 Continue\r\n": false,
+		"HTTP/1.0 10r Continue\r\n": false,
+		"HTTP/1.1 200\r\n":          false,
```

Comment:
can you add this test case `HTTP/1.1 200\r\nServer: Tengine\r\nContent-Length: 0\r\nConnection: close\r\n`?

the current implementation does not ok with the input `HTTP/1.1 200\r\n` but will panic with `HTTP/1.1 200\r\nServer: Tengine\r\nContent-Length: 0\r\nConnection: close\r\n`

Reply:
I added the test case and it has passed!